From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Thu, 30 Apr 2020 12:17:18 +0200
Subject: ANDROID: driver: gpu: drm: fix export symbol types

In commit 310dd4fe8350 ("ANDROID: driver: gpu: drm: add notifier for
panel related events") a number of new symbols were exported, but they
should have been set as a _GPL symbol.

Fix this up by properly changing the export types.

Bug: 139653858
Cc: Shashank Babu Chinta Venkata <sbchin@codeaurora.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I54dbc1f4d41739ab19040563966a6505897734fb
---
 drivers/gpu/drm/drm_panel.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/drm_panel.c b/drivers/gpu/drm/drm_panel.c
--- a/drivers/gpu/drm/drm_panel.c
+++ b/drivers/gpu/drm/drm_panel.c
@@ -347,21 +347,21 @@ int drm_panel_notifier_register(struct drm_panel *panel,
 {
 	return blocking_notifier_chain_register(&panel->nh, nb);
 }
-EXPORT_SYMBOL(drm_panel_notifier_register);
+EXPORT_SYMBOL_GPL(drm_panel_notifier_register);
 
 int drm_panel_notifier_unregister(struct drm_panel *panel,
 	struct notifier_block *nb)
 {
 	return blocking_notifier_chain_unregister(&panel->nh, nb);
 }
-EXPORT_SYMBOL(drm_panel_notifier_unregister);
+EXPORT_SYMBOL_GPL(drm_panel_notifier_unregister);
 
 int drm_panel_notifier_call_chain(struct drm_panel *panel,
 	unsigned long val, void *v)
 {
 	return blocking_notifier_call_chain(&panel->nh, val, v);
 }
-EXPORT_SYMBOL(drm_panel_notifier_call_chain);
+EXPORT_SYMBOL_GPL(drm_panel_notifier_call_chain);
 
 MODULE_AUTHOR("Thierry Reding <treding@nvidia.com>");
 MODULE_DESCRIPTION("DRM panel infrastructure");
